docs/modules/ROOT/pages/documentation/extensions/autoscale/README.adoc

```diff
@@ -0,0 +1,110 @@
+= AutoScale Extensions
+
+The AutoScale extensions provide the means for users to scale Payara Server instances up and down
+across different environments.
+
+You can find the code for each AutoScale extension here: https://github.com/payara/AutoScale-Groups. The codebase includes a notifier example in case a notifier you want is not available and you'd like to implement it yourself.
+
+== AutoScale Summary
+
+AutoScale is split into a number of separate modules, but can be summarised into:
+
+* An API module
+* Two Core modules (base implementation module and a base Admin Console Plugin)
+* Extension modules
+
+### The API Module
+This module contains the interfaces and abstract classes intended for use or extension within Payara Server and any
+extensions. More specifically, it contains:
+
+* The "group" `ScalingGroups` ConfigBean interface (representing the domain.xml elements and attributes) - this
+contains the list of all configured scaling groups
+* The "base" `ScalingGroup` ConfigBean interface - This is the ConfigBean interface any extension ConfigBean interfaces
+should extend from to create their own scaling group type.
+* The `Scaler` abstract class contract - This is the abstract class extensions should create an implementation service
+for. This service is used to perform the `scaleUp` and `scaleDown` operations.
+* The `Scales` qualifier - This qualifier is used to link a `Scaler` service to a specific `ScalingGroup`, there is
+only ever meant to be one `Scaler` service for each `ScalingGroup` extension.
+* Abstract classes for creating the get, set, and create Admin Commands from.
+
+This module is expected to be included in the server, regardless of what extension you choose to use.
+
+### The Core Modules
+There are currently two core modules: a base implementation module, and a base Admin Console plugin.
+
+Both of these modules are expected to be included in the server, regardless of what extension you choose to use.
+
+#### The Base Implementation Module
+This module contains generic admin commands and util classes:
+* The `scale-up` and `scale-down` admin commands - this command looks up the `Scaler` service for the `ScalingGroup`
```

Comment:
Please add a blank line before this line to fix the formatting. Lists need to have a blank line above them otherwise they are ignored in AsciiDoc.